### PR TITLE
Improves #2440 - Unit test for ContinueOnErrorStep

### PR DESCRIPTION
**Pull request by kolloch**
also move Mockito test support to test package.

More tests to follow...

### CODE REVIEW

File: src/test/scala/mesosphere/marathon/core/task/tracker/impl/steps/ContinueOnErrorTest.scala [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    When("executing the step")
+    val logEvents = CaptureLogEvents.forBlock {
+      val resultFuture = processUpdate(ContinueOnErrorStep(nested))
+      Await.result(resultFuture, 3.seconds)
+    }
+
+    Then("it should execute the nested step")
+    processUpdate(verify(nested, times(1)))
+    And("not produce any logging output")
```
Comment (meichstedt) [inline src/test/scala/mesosphere/marathon/core/task/tracker/impl/steps/ContinueOnErrorTest.scala]: s/not produce any/produce

**Comment by meichstedt** [comment]
Just one little comment.

File: src/test/scala/mesosphere/marathon/core/task/tracker/impl/steps/ContinueOnErrorTest.scala (hunk 1 above)
Comment (kolloch) [inline src/test/scala/mesosphere/marathon/core/task/tracker/impl/steps/ContinueOnErrorTest.scala]: ack

**Comment by kolloch** [comment]
I changed the text of the "And" explanation and renamed the test from ContinueOnErrorTest to ContinueOnError_Step_Test

**Comment by meichstedt** [comment]
Thanks!

**Pull request merged**